File: application/models/Nommat.php
<?php

class Application_Model_Nommat extends Application_Model_Element_Abstract {
    protected $_idField = 'nomid';
    protected $_matid;
    protected $_matqte;
    
    public function getNomid() {
        return $this->_nomid;
    }

    public function setNomlib($nomid) {
        $this->_nomid = $nomid;
    }

    public function getMatid() {
        return $this->_matid;
    }

    public function setMatid($_matid) {
        $this->_matid = $_matid;
    }

    public function getMatqte() {
        return $this->_matqte;
    }

    public function setMatqte($_matqte) {
        $this->_matqte = $_matqte;
    }


    
}

?>

File: src/models/Nommat.java
/*
 * To change this template, choose Tools | Templates
 * and open the template in the editor.
 */
package models;

import javax.persistence.Column;
import javax.persistence.Entity;
import javax.persistence.GeneratedValue;
import javax.persistence.GenerationType;
import javax.persistence.Id;
import javax.persistence.Table;

/**
 *
 * @author overadmin
 */
@Entity
@Table(name="nommat")
public class Nommat {
    
    @Id
    @GeneratedValue(strategy= GenerationType.IDENTITY)
    private int nomid;   
     
    @Column(name="matid", columnDefinition="INTEGER", nullable=true)
    private int matid;
    
    @Column(name="matqte", columnDefinition="INTEGER", nullable=true)
    private int matqte;

    public int getNomid() {
        return nomid;
    }

    public void setNomid(int nomid) {
        this.nomid = nomid;
    }

    public int getMatid() {
        return matid;
    }

    public void setMatid(int matid) {
        this.matid = matid;
    }

    public int getMatqte() {
        return matqte;
    }

    public void setMatqte(int matqte) {
        this.matqte = matqte;
    }   
}
